source/crud/tests/unified/findOneAndDelete-let.yml

```diff
@@ -100,8 +65,8 @@ tests:
           let: &let1
             id: 1
         expectError:
-          # Older server versions may not report an error code, but the error
-          # message is consistent between 2.6.x and 4.4.x server versions.
+          # This error message is consistent between for 4.4+ servers.
+          # Older servers return a different error message.
           errorContains: "field 'let' is an unknown field"
```

Comment:
Separate question: since you're requiring 3.6+ in other tests and 4.2+ here, are those ranges of server versions consistent that we can just use an error code assertion?

Reply:
Apparently, for delete and update all versions down to 3.6 return same error message, while for findAndModify there is a difference between [3.6, 4.2) and [4.2, 4.4].
4.4: `"errMsg":"BSON field 'let' is an unknown field.","errName":"Location51177", "errCode":51177`
4.0: `errMsg:"Use of undefined variable: id" errName:Location17276 errCode:17276`